src/comp_loinc/loinc_builder_steps.py

```diff
@@ -141,6 +141,7 @@ def loinc_parts_all(
             ),
         ] = False,
     ):
+        logger.info(f"Starting lp-inst-all")
```

Comment:
## Log on all methods, or just some?
And if just some, are any methods missing logging that should?

Context: I was troubleshooting and for me it would've been helpful to see more logging, though it could be `DEBUG`; doesn't have to be `INFO`.